src/_sass/components/_toc.scss

```diff
@@ -4,6 +4,15 @@
   display: none;
   right: 0;
 
+
+  @mixin nav-link-not-active-hover {
+    &:not(.disabled) {
```

Comment:
I'm not sure the TOC entries can be disabled, at least I can't find any area where they can be. Do you know of any? If not, we can likely remove the `&:not(.disabled) {` portion.

Reply:
You is correctly, I only copy and paste from another place.